SPECS-EXTENDED/proj/proj.spec

```diff
@@ -0,0 +1,959 @@
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+%define data_version 1.14
+%define sover   25
+%define libname lib%{name}%{sover}
+Summary:        Cartographic projection software
+Name:           proj
+Version:        9.2.1
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://proj.org/
+Source0:        https://download.osgeo.org/proj/%{name}-%{version}.tar.gz
+Source1:        https://download.osgeo.org/%{name}/%{name}-data-%{data_version}.tar.gz
+BuildRequires:  cmake
+BuildRequires:  gcc-c++
+BuildRequires:  gmock-devel
+BuildRequires:  gtest-devel
+BuildRequires:  pkgconfig
+BuildRequires:  sqlite
+BuildRequires:  pkgconfig(libcurl)
+BuildRequires:  pkgconfig(libtiff-4)
+BuildRequires:  pkgconfig(nlohmann_json)
+BuildRequires:  pkgconfig(sqlite3)
+Provides:       libproj = %{version}
+
+%description
+This package offers the commandline tools for performing respective
+forward and inverse transformation of cartographic data to or from cartesian
+data with a wide range of selectable projection functions.
+
+%package -n %{libname}
+Summary:        Cartographic projection software
+License:        MIT
+
+%description -n %{libname}
+This package the library for performing respective
+forward and inverse transformation of cartographic data to or from cartesian
+data with a wide range of selectable projection functions.
+
+%package devel
+Summary:        Development files for PROJ
+License:        MIT
+Requires:       %{libname} = %{version}
+Provides:       libproj-devel = %{version}
+Obsoletes:      libproj-devel < %{version}
+
+%description devel
+This package contains libproj and the appropriate header files and man pages.
+
+%define data_subpkg(c:n:e:s:) \
+%define countrycode %{-c:%{-c*}}%{!-c:%{error:Country code not defined}} \
+%define countryname %{-n:%{-n*}}%{!-n:%{error:Country name not defined}} \
+%define extrafile %{-e:%{_datadir}/%{name}/%{-e*}} \
+%define wildcard %{!-s:%{_datadir}/%{name}/%{countrycode}_*} \
+\
+%package data-%{countrycode}\
+Summary:      %{countryname} datum grids for Proj\
+BuildArch:    noarch\
+# See README.DATA \
+License:      BSD-2-Clause AND CC0-1.0 AND CC-BY-4.0 AND CC-BY-SA-4.0 AND SUSE-Public-Domain \
+Supplements:  proj\
+\
+%description data-%{countrycode}\
+%{countryname} datum grids for Proj.\
+\
+%files data-%{countrycode}\
+%{wildcard}\
+%{extrafile}
+License:        MIT
```

Comment:
I think we already have `License` on line 23

Reply:
Yes. It is there at the top of the spec file. I have removed the redundant ones.